applications/luci-app-socatg/root/etc/init.d/socatg

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh /etc/rc.common
```

Comment:
If you integrate the new uci options into the socat package, then this should not be necessary.

https://github.com/openwrt/packages/blob/master/net/socat/files/socat.init